### ISSUE TITLE
[%collapsible] and [%autofit] are not described in the manual

### ISSUE THREAD

**Issue by silvester747** [asciidoctor/asciidoctor#3654]
Through various blogposts I learned about collapsible blocks with [%collapsible] and autofitting source code with [%autofit]. However, I cannot seem to find any reference to this in the manual (https://asciidoctor.org/docs/user-manual).

Should this become part of the manual? Or is there some other place where this functionality is described?

**Comment by mojavelinux** [comment]
Yes, it should be part of the manual. We're in the process of migrating the docs to a new system (Antora), which is why it's lagging a bit right now.

I'm transferring this issue to where the docs are currently located.
